api-server/openapi.yml

```diff
@@ -13,12 +13,12 @@ info:
     to get access to the [management api](https://github.com/apache/activemq-artemis/blob/main/docs/user-manual/management.adoc#the-management-api) of a broker instance via 
     its [jolokia](https://jolokia.org/) endpoint.
 
-    ### How to add new apis to the api-server
+    ### How to add new apis to the src
 
     The api server uses [openapi](https://www.openapis.org/) to define
-    its apis. All the apis are defined in the file [openapi.yml](api-server/config/openapi.yml).
+    its apis. All the apis are defined in the file [openapi.yml](src/config/openapi.yml).
```

Comment:
@lavocatt , in the codebase the api-server path is still pointing to `api-server/openapi.yml` instead `src/config/openapi.yml`. could you please update this path in the src.